teacher.rb

```diff
@@ -0,0 +1,50 @@
+class Teacher
+  attr_reader :age, :phase, :target_raise, :salary
+  attr_accessor :name
```

Comment:
Some of these look awfully similar to things in Student... how could you refactor?